recipes/jasper/all/conanfile.py

```diff
@@ -129,19 +105,19 @@ def _create_cmake_module_variables(self, module_file):
 
     @property
     def _module_file_rel_path(self):
-        return os.path.join("lib", "cmake", "conan-official-{}-variables.cmake".format(self.name))
+        return os.path.join("lib", "cmake", f"conan-official-{self.name}-variables.cmake")
 
     def package_info(self):
         self.cpp_info.set_property("cmake_find_mode", "both")
         self.cpp_info.set_property("cmake_file_name", "Jasper")
         self.cpp_info.set_property("cmake_target_name", "Jasper::Jasper")
         self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])
         self.cpp_info.set_property("pkg_config_name", "jasper")
+        self.cpp_info.libs = ["jasper"]
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.extend(["m", "pthread"])
```

Comment:
`pthread` is a special library - gcc has a flag for it (looking into this..)

Reply:
It's a private dependendency I think, no need for the flag I think. This block could even be omitted when jasper is shared.